src/components/wizard/wizard-test-wrapper.ts

```diff
@@ -43,14 +43,16 @@ export class WizardTestWrapper<T> {
         return buttons.find(button => button.innerText.toUpperCase() === buttonText.toUpperCase());
     }
 
-    async clickStepButton(buttonText: string): Promise<void> {
+    async clickStepButton(buttonText: string, detectChanges: boolean = true): Promise<void> {
         const button = this.getStepButton(buttonText);
         if (!button) {
             throw new Error(`Button "${buttonText}" not found`);
         }
 
         button.click();
-        this._fixture.detectChanges();
+        if (detectChanges) {
+            this._fixture.detectChanges();
+        }
```

Comment:
Not sure about this, the detectChanges here seems to emit the stepChanges twice. This isn't an issue in the docs or plunkers as well as clicking on the next button in the karma test when running it locally. Any help on this would be appreciated

Reply:
In the plunker linked in the ticket, stepChange emits twice when you click Next. So this test is probably accurate.